modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+---
+
+=== InfluxDB
+
+All InfluxDB data/meta nodes run on the 3 nodes labeled `amv_node` and therefore we need to backup the data sitting in those 3 nodes. Data is under '/var/lib/data/influxdb' for both meta and data pods. The following steps consider that the underlying hard drive has enough space to hold the backup. The destination can be actually anything (disk, S3 bucket, etc).
+
+. `kubectl get nodes | grep amv` will provide the Anypoint nodes
+. Connect to each `amv` node and run `sudo gravity enter`
+. Under `cd /var/lib/data/influxdb` we can find the data that needs to be backup. At this point, any approach to copy that data to a secure location is acceptable. The following steps are provided just as a local backup.
+. Copy over data with `cp -a meta/. bck/meta & cp -a data/. bck/data` (`bck` is just a directory in the same location, it can be anywhere)
+. On a fresh new installation with InfluxDB pods already running we will restore data with `cp -a bck/data/. data/ & cp -a bck/meta/. meta/`
+. Restarting all meta/data pods will reload that data
+
+
+Backing up InfluxDB is as simple as copying over all the data under `/var/lib/data/influxdb`. Upon restore, all influxDB pods need to be restarted (if they are already running) to restore the data.
+
+=== Cassandra
+
+The `dias-prov-k8s-cassandra-comp` represents the cassandra cluster required by Insights. To create a snapshot of the cassandra cluster, we need to follow the below steps:
+
+. Make sure you have copied any previous backups generated. The next step will override any existing backup created
+. There are 3 Cassandra Pods. We need to back-up the data from each pod separately
+  . For Pod-0: `kubectl -n dias exec -it dias-prov-k8s-cassandra-comp-0 -c cassandra ./cassandra-backup.sh`
+  . For Pod-1: `kubectl -n dias exec -it dias-prov-k8s-cassandra-comp-1 -c cassandra ./cassandra-backup.sh`
+  . For Pod-2: `kubectl -n dias exec -it dias-prov-k8s-cassandra-comp-2 -c cassandra ./cassandra-backup.sh`
+. The backup gets created on each of the `amv` nodes. There should be 3 `amv` nodes. `kubectl get nodes | grep amv` will provide the Anypoint nodes
+. Connect to each `amv` node and run `sudo gravity enter`
+. Under `cd /var/lib/data/dias-cassandra/backup` we can find the data that needs to be backup. At this point, any approach to copy that data to a secure location is acceptable. Please, move the data out of this location else, it will be over-written by the next backup operation.
+
+=== Logstash
+
+When it comes to Logstash data (`dias-prov-k8s-am-ingestor-comp`) it is important to consider that Logstash keeps "watermark" files which indicate which metric files have been processed. In a fresh new instal (either from scratch or after a restore) there will not be any metric files to process and therefore no need to backup those "watermark" files. This is acceptable since Logstash has a flush mechanism built-in where upon pod graceful termination, all data is flushed to InfluxDB.
```

Comment:
Let's not include this section if there's no need to do anything here

Reply:
Done!